packages/diagrams/frontend/sirius-components-diagrams/src/renderer/connector/ConnectorContextualMenu.types.ts

```diff
@@ -112,4 +112,5 @@ export interface GQLErrorPayload extends GQLInvokeSingleClickOnTwoDiagramElement
 
 export interface GQLSuccessPayload extends GQLInvokeSingleClickOnTwoDiagramElementsToolPayload {
   messages: GQLMessage[];
+  newSelection: GQLWorkbenchSelection;
```

Comment:
The type is probably `GQLInvokeSingleClickOnTwoDiagramElementsToolSuccessPayload` and not `GQLSuccessPayload`